Automattic/wp-calypso
# Reading Settings: Adding 'Blogs post' setting to the new Reading Settings page

#### Proposed Changes

* Add **Blogs post** setting to the new Reading Settings page

#### Testing Instructions



* Run local version of calypso
* Go to `http://calypso.localhost:3000/settings/reading/<YOUR-BLOG>`
* Ensure the setting: 
  * displays the correct value
  * can be modified and saved
* Validate that the modified value actually affects how many posts are displayed on a blog posts page
* Smile

#### Pre-merge Checklist





Closes #70410

![Screen Shot 2022-12-08 at 15 30 06](https://user-images.githubusercontent.com/2019970/206472563-0602a942-09ea-49be-8581-3fe78ef80fc9.png)
